style.css

```diff
@@ -0,0 +1,116 @@
+.container {
+    display: flex;
+    flex-direction: column;
+}
+
+.searching {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    height: 60px;
+    background-color: #5362CE;
+}
+
+.logo-circle {
+    margin-left: 15px;
+    width: 45px;
+    height: 45px;
+    border-radius: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.logo-text {
+    font-weight: bolder;
+    color: white;
+
+}
+
+.search-bar {
+    display: flex;
+    justify-content: left;
+    padding-left: 20px; 
+    background-color: white;
+    width: 380px;
+    height: 40px;
+    border: 0px;
+    border-radius: 20px;
+}
+
+.search-bar::placeholder {
+    color: #d1d1d1;
```

Comment:
Please specify all the colors as variables using double dash aka custom property in css.